src/pudl_archiver/depositors/fsspec.py

```diff
@@ -180,10 +181,20 @@ def get_checksum(self, filename: str) -> str | None:
         Args:
             filename: Name of file to checksum.
         """
+        md5_hash = None
         filepath = self.deposition.deposition_path / filename
         if filepath.exists():
-            return compute_md5(filepath)
-        return None
+            # For Google Cloud fsspec backend we can use the `info` method to get
```

Comment:
I hadn't realized this, but there's actually a somewhat recent archiver change that already implements a checksum check on uploaded files, and will retry the upload if it fails, so I don't need to make that change. However, I also realized that the way we've been computing the checksum means that while using the `fsspec` storage backend, we would first upload the file, then read the file contents from the upload location to compute a new checksum. This means we would have to download the file from GCS when we do this computation. I've changed this now, so it will just ask GCS for the checksum, which is significantly faster and should reduce data egress.